ui/apps/platform/src/Containers/Integrations/utils/integrationsList.ts

```diff
@@ -25,14 +25,14 @@ import pagerduty from 'images/pagerduty.svg';
 import tenable from 'images/tenable.svg';
 import signature from 'images/signature.svg';
 
-// Adding an integration tile behind a feature flag
-// To add a new integration, uncomment the following import
-
-// import { knownBackendFlags } from 'utils/featureFlags';
-
-// and then add the following property to the new tiles object definition in the list below:
-//     featureFlagDependency: knownBackendFlags.ROX_<flag_constant>,
+/*
+ * To add an integration tile behind a feature flag:
```

Comment:
Nice, I like the addition of comments for clear instructions on how to add a new one